src/main/java/mat/model/cql/CQLLibraryDataSetObject.java

```diff
@@ -39,7 +39,18 @@
 	private boolean isVersionable;
 	private boolean isDeletable;
 	private String libraryModelType;
-	
+	private boolean isCheckboxSelected = false;

```

Comment:
I know we talked about this approach but it doesn't look like you are utilizing it currently. If you think you'll need it for the final solution, lets leave it! Otherwise we should remove it now so we don't accidentally keep it in.

Reply:
I will make a note to remove it in future commit, If not being used.